### ISSUE TITLE
tns doctor => cocoapods failed

### ISSUE THREAD

**Issue by njl07** [NativeScript/nativescript-cli#1741]
Hello,

When try to do `tns doctor`, I have error with `Podfile` of `AFNetworking`:

```bash
Verifying CocoaPods. This may take more than a minute, please be patient.
  ◝ Installing iOS runtime.(node:85055) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
  ◝ Installing iOS runtime.tns-ios@2.0.0 ../../var/folders/br/8v_dvjf57v33lg60t_711qh00000gn/T/nativescript-check-cocoapods11646-85055-xtr4je.b2w8ode7b9/node_modules/tns-ios
  ◠ Verifying CocoaPods. This may take some time, please be patient..
Analyzing dependencies
[!] The dependency `AFNetworking (~> 1.0)` is not used in any concrete target.
 
WARNING: There was a problem with CocoaPods
Verify that CocoaPods are configured properly.
There seem to be issues with your configuration.
```

I've reinstalled so many times `cocoapods` in `v0.39` and in `pre` release but nothing works.

Can you help me?

Thanks.

**Comment by imjohnbo** [comment]
I have this problem too.

Any ideas?

**Comment by ilpaijin** [comment]
the same

**Comment by apntzls** [comment]
Ditto

**Comment by preekmr** [comment]
Same problem here! 

`Verifying CocoaPods. This may take more than a minute, please be patient.
  ◡ Installing iOS runtime.tns-ios@2.0.0 ../../var/folders/gn/_vqlg02n1r3_3pb_csvyrqgc0000gn/T/nativescript-check-cocoapods116415-47389-15druqb/node_modules/tns-ios
  ◡ Verifying CocoaPods. This may take some time, please be patient..
Analyzing dependencies
[!] The dependency `AFNetworking (~> 1.0)` is not used in any concrete target.
 
WARNING: There was a problem with CocoaPods
Verify that CocoaPods are configured properly.
There seem to be issues with your configuration.
`

**Comment by ejsuncy** [comment]
same here. I've been stuck on this part of the chapter one tutorial for dayz...

**Comment by preekmr** [comment]
Hey @ejsuncy, though I am getting the error I could able to run the app in the IOS emulator. Probably you can check on that.

**Comment by fryck** [comment]
Same for me

**Comment by snailbites** [comment]
+1

**Comment by Daxito** [comment]
+1

**Comment by bendav91** [comment]
Also having the same problem. On Running `$ tns doctor` after the dependency installation, I'm getting ``[!] The dependency `AFNetworking (~> 1.0)` is not used in any concrete target.``

**Comment by ejsuncy** [comment]
I had skipped installing node because I already had it. I ended up installing node 4.4.4 from the website (not with homebrew), which also updated npm to 2.15.1. I then started the tutorial over again, installing nativescript, and everything works as expected now...problem solved for me :)

**Comment by Mitko-Kerezov** [comment]
This issue was resolved with the latest `2.0.1` version of the NativeScript CLI

**Comment by gatspy** [comment]
2.1.1, Ditto

**Comment by sittingbool** [comment]
Same here, Version 2.2.1. Is it a Pods problem? Does reinstalling help?

**Comment by justinrassier** [comment]
Version 2.2.1. I gave reinstalling a try and it didn't seem to help.

**Comment by justinrassier** [comment]
This is the command it seems to be blowing up at:

`Errno::EACCES - Permission denied @ dir_s_mkdir - /Users/justin.rassier/Library/Caches/CocoaPods`

**Comment by NicoleZA** [comment]
I also cannot install cocoa pods.
I have the latest nativescript cli

**Comment by florianbepunkt** [comment]
Same problem here: I installed nativescript 2.3.0 and followed your getting started guide. tns doctor fails with 
```

WARNING: There was a problem with CocoaPods
Verify that CocoaPods are configured properly.
There seem to be issues with your configuration.
```

**Comment by sudhanshu-15** [comment]
NativeScript uses the sandbox-pod executable. This may prevent some pods from installing correctly. If you encounter such cases, you can switch to the regular pod executable:
+>
+> 1. Open the NativeScript CLI configuration file, usually located in /usr/local/lib/node_modules/nativescript/config/config.json.
+> 1. Change the value of USE_POD_SANDBOX to false.

This worked for me.

https://github.com/NativeScript/docs/pull/259/files

**Comment by rosen-vladimirov** [comment]
@sudhanshu-15 , @florianbepunkt , please make sure you have Cocoapods 1.0.0 or later installed on your machines. Due to latest changes in Cocoapods (described in their [blog](http://blog.cocoapods.org/Sharding/)) versions pre 1.0.0 will not work.

**Comment by sudhanshu-15** [comment]
@rosen-vladimirov this is a fix for Cocoapods 1.0.1 and 1.1.1
